pkg/tsdbctl/query.go

```diff
@@ -66,15 +86,22 @@ func newQueryCommandeer(rootCommandeer *RootCommandeer) *queryCommandeer {
 		},
 	}
 
-	cmd.Flags().StringVarP(&commandeer.to, "end", "e", "", "to time")
-	cmd.Flags().StringVarP(&commandeer.from, "begin", "b", "", "from time")
-	cmd.Flags().StringVarP(&commandeer.output, "output", "o", "", "output format: text,csv,json")
-	cmd.Flags().StringVarP(&commandeer.filter, "filter", "f", "", "v3io query filter e.g. method=='get'")
-	cmd.Flags().StringVarP(&commandeer.last, "last", "l", "", "last min/hours/days e.g. 15m")
-	cmd.Flags().StringVarP(&commandeer.windows, "windows", "w", "", "comma separated list of overlapping windows")
+	cmd.Flags().StringVarP(&commandeer.to, "end", "e", "",
+		"End (maximum) time for the query, as a string containing an\nRFC3339 time string, a Unix timestamp in milliseconds, or\na relative time of the format \"now\" or \"now-[0-9]+[mhd]\"\n(where 'm' = minutes, 'h' = hours, and 'd' = \"days\").\nExamples: \"2018-09-26T14:10:20Z\"; \"1537971006000\";\n\"now-3h\"; \"now-7d\". (default \"now\")")
+	cmd.Flags().StringVarP(&commandeer.from, "begin", "b", "",
+		"Start (minimum) time for the query, as a string containing\nan RFC3339 time, a Unix timestamp in milliseconds, a\nrelative time of the format \"now\" or \"now-[0-9]+[mhd]\"\n(where 'm' = minutes, 'h' = hours, and 'd' = \"days\"), or 0\nfor the earliest time. Examples: \"2016-01-02T15:34:26Z\";\n\"1451748866\"; \"now-90m\"; \"0\". (default = <end time> - 1h)")
+	cmd.Flags().StringVarP(&commandeer.output, "output", "o", formatter.DefaultOutputFormat,
```

Comment:
Why did you change the "default" value?

Reply:
I'm now sure which change you're referring to. The original code didn't document the default values for the `query` end and start time flags. I first added documentation of the default value for the `-e|--end` flag (commit 15fd9b4) and then for the `'-b|--begin` flag (commit e64d18e).